packages/rlc-common/src/buildTopLevelIndexFile.ts

```diff
@@ -44,7 +44,7 @@ export function buildTopLevelIndex(model: RLCModel) {
     });
     const content = indexFile.getFullText();
     const filePath = path.join(
-      srcPath.substring(0, srcPath.indexOf("/src/") + 5),
+      srcPath.substring(0, srcPath.indexOf(path.sep + "src" + path.sep) + 5),
```

Comment:
Could you also change the logic here?
https://github.com/Azure/autorest.typescript/blob/main/packages/rlc-common/src/helpers/pathUtils.ts#L3